### PR TITLE
[12.0][MIG] account_cash_invoice

### PR DESCRIPTION
**Pull request by jarroyomorales**
Migration to 12.0 of account_cash_invoice.
Superseeded #240 keeping @mamcode commits and attended @kittiu comments.

### CODE REVIEW

**Comment by etobella** [comment]
@mamcode @kittiu @pedrobaeza This superseeds #240 and all comments have been attended

**Review by etobella — APPROVED** [review]

File: account_cash_invoice/__manifest__.py
Comment (kittiu) [inline account_cash_invoice/__manifest__.py]: Should we always use 12.0.1.0.0 for the first version?
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration

**Review by kittiu — APPROVED** [review]

File: account_cash_invoice/__manifest__.py
Comment (jarroyomorales) [reply account_cash_invoice/__manifest__.py]: True, just changed it!

**Comment by etobella** [comment]
@jbeficent @pedrobaeza some PR are blocked by this one, there is 2 approvals and comes from another PR where there was an extra approval. Could we merge this one?

**Pull request closed without merging**
